AABB[] aabbs

# Capsule[] capsules

# Cone[] cones

# Cylinder[] cylinders

# Ellipsoid[] ellipsoids

Frustum[] frustums

LineSegment[] line_segments

OBB[] obbs

Plane[] planes

Point[] points

Ray[] rays

Sphere[] spheres

# Triangle[] triangles